qm-disk-resize: add page; qm-resize: add alias page (#10496)
tldr-pages/tldr: pages/linux/qm-disk-resize.md

# qm disk resize

> > Resize a virtual machine (VM) in the Proxmox Virtual Environment (PVE).

Reviewer: ```suggestion
> Resize a virtual machine disk in the Proxmox Virtual Environment (PVE).
```

> # qm-disk-resize
> 
> > This command is an alias of `qm-resize`.
> > More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
> 
> - View documentation for the original command:
> 
> `tldr qm-resize`

Reviewer 2: ```suggestion
# qm disk resize

> Resize a virtual machine (VM) in the Proxmox Virtual Environment (PVE).
> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.

- Resize a virtual machine by id:

`qm disk resize {{vm_id}} {{disk_name}} +{{resize_value}}G`
```

> - Resize a virtual machine by id:
> 
> `qm disk resize {{vm_id}} {{disk_name}} +{{resize_value}}G`

Reviewer: ```suggestion
- Add `n` gigabytes to a virtual disk:

`qm disk resize {{vm_id}} {{disk_name}} +{{n}}G`
```
